msgid ""
msgstr ""
"Project-Id-Version: src_debug_tserver\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-03-21 10:17+EDT\n"
"PO-Revision-Date: 2018-03-18 22:27+0300\n"
"Last-Translator: alexandr\n"
"Language-Team: Russian\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10 >= 2 && n"
"%10<=4 &&(n%100<10||n%100 >= 20)? 1 : 2);\n"
"X-Generator: Loco https://localise.biz\n"
"X-Loco-Source-Locale: en\n"
"X-Loco-Project-Id: 44387\n"
"X-Loco-Api-Version: 1.0.19 20180317-1\n"

#: src/debug/tserver/abstract.py:331
msgid "Wing Debug Log File\n"
msgstr "Файл журнала отладки Wing"

#: src/debug/tserver/abstract.py:332
msgid "This file can be used to diagnose problems in debugger internals\n"
msgstr ""
"Этот файл можно использовать для диагностики проблем во внутренних "
"компонентах отладчика"

#: src/debug/tserver/abstract.py:333
msgid "For help, contact support@wingware.com\n"
msgstr "За помощью обращайтесь по адресу support@wingware.com"

#: src/debug/tserver/abstract.py:338
msgid "Warning:  Log file does not exist:\n"
msgstr "Предупреждение: файл журнала не существует:"

#: src/debug/tserver/abstract.py:340
msgid "Warning:  and could not be created; logging disabled\n"
msgstr "Предупреждение: и не удалось создать; ведение журнала отключено"

#: src/debug/tserver/abstract.py:343
msgid "Warning:  Log file is not a file:\n"
msgstr "Предупреждение: файл журнала не является файлом:"

#: src/debug/tserver/abstract.py:345
msgid "Warning:  Logging disabled\n"
msgstr "Предупреждение: ведение журнала отключено"

#: src/debug/tserver/dbginspect.py:54 src/debug/tserver/dbgserver.py:2906
#: src/debug/tserver/dbgutils.py:443 src/debug/tserver/dbgutils.py:463
msgid "(truncated)"
msgstr ""

#: src/debug/tserver/dbginspect.py:920 src/debug/tserver/dbgutils.py:451
#: src/debug/tserver/dbgutils.py:473 src/debug/tserver/dbgutils.py:2949
msgid "<opaque "
msgstr "<непрозрачный "

#: src/debug/tserver/dbgserver.py:1176
msgid "<module>"
msgstr "<модуль>"

#: src/debug/tserver/dbgserver.py:1324
msgid "<error evaluating>"
msgstr "<оценка ошибок>"

#: src/debug/tserver/dbgserver.py:1897 src/debug/tserver/dbgserver.py:1935
msgid "<thread %i is running>"
msgstr "<поток %i запущен>"

#: src/debug/tserver/dbgserver.py:1900 src/debug/tserver/dbgserver.py:1938
msgid "<invalid thread id %s or stack index %s>"
msgstr "<Неверный поток id %s или указатель стека  %s>"

#: src/debug/tserver/dbgserver.py:1902 src/debug/tserver/dbgserver.py:1940
msgid "<invalid stack frame>"
msgstr "<недопустимый фрейм стека>"

#: src/debug/tserver/dbgserver.py:1968 src/debug/tserver/dbgserver.py:1997
msgid "<sys.modules does not exist>"
msgstr "<sys.modules не существует>"

#: src/debug/tserver/dbgserver.py:1971 src/debug/tserver/dbgserver.py:2003
msgid "<error accessing sys.modules>"
msgstr "<ошибка доступа sys.modules>"

#: src/debug/tserver/dbgserver.py:2000
msgid "<no longer exists"
msgstr "<больше не существует"

#: src/debug/tserver/dbgserver.py:2048 src/debug/tserver/dbgserver.py:2101
msgid "<no longer exists>"
msgstr "<более не существует>"

#: src/debug/tserver/dbgserver.py:2079 src/debug/tserver/dbgserver.py:4106
msgid "<parent too large>"
msgstr "<слишком большое значение родителя>"

#: src/debug/tserver/dbgserver.py:2081 src/debug/tserver/dbgserver.py:4108
msgid "<invalid parent type>"
msgstr "<неверный тип родителя>"

#: src/debug/tserver/dbgserver.py:2091 src/debug/tserver/dbgserver.py:2543
msgid "<value not found>"
msgstr "<значение не найдено>"

#: src/debug/tserver/dbgserver.py:2531
msgid "<thread is running>"
msgstr "<поток запущен>"

#: src/debug/tserver/dbgserver.py:2533
msgid "<invalid context>"
msgstr "<недопустимый контекст>"

#: src/debug/tserver/dbgserver.py:4114
msgid "<invalid path %s at pos=%i>"
msgstr "<неверный путь %s в pos=%i>"

#: src/debug/tserver/dbgutils.py:150
msgid "<undefined>"
msgstr ""

#: src/debug/tserver/dbgutils.py:166
msgid "at"
msgstr ""

#: src/debug/tserver/dbgutils.py:174
msgid "dim=%sx%s>"
msgstr ""

#: src/debug/tserver/dbgutils.py:176
msgid "len=%s>"
msgstr ""

#: src/debug/tserver/dbgutils.py:186 src/debug/tserver/dbgutils.py:2451
#, fuzzy
msgid "<error handling value>"
msgstr "<оценка ошибок>"

#: src/debug/tserver/dbgutils.py:188
msgid "<timeout>"
msgstr ""

#: src/debug/tserver/dbgutils.py:196
msgid "<huge "
msgstr ""

#: src/debug/tserver/dbgutils.py:216
msgid "<long "
msgstr ""

#: src/debug/tserver/dbgutils.py:233
#, fuzzy
msgid "<opaque"
msgstr "<непрозрачный "

#: src/debug/tserver/dbgutils.py:465
msgid "<huge %s at %s; len=%i>"
msgstr "<очень большой %s в %s; len=%i>"

#: src/debug/tserver/dbgutils.py:653 src/debug/tserver/dbgutils.py:665
msgid ", pos "
msgstr ", поз "

#: src/debug/tserver/dbgutils.py:669
msgid ", line "
msgstr ", строка "

#: src/debug/tserver/dbgutils.py:671
msgid "SyntaxError: "
msgstr "Ошибка синтаксиса: "

#: src/debug/tserver/dbgutils.py:699
msgid "Error"
msgstr "Ошибка"

#: src/debug/tserver/dbgutils.py:707
msgid "Error: Wing was unable to pack up an exception message"
msgstr "Ошибка: Wing не смог собрать сообщение об исключении"

#: src/debug/tserver/dbgutils.py:3258 src/debug/tserver/dbgutils.py:3274
msgid "Key"
msgstr ""

#: src/debug/tserver/dbgutils.py:3258 src/debug/tserver/dbgutils.py:3274
msgid "Value"
msgstr ""

#: src/debug/tserver/dbgutils.py:3269 src/debug/tserver/dbgutils.py:3296
msgid "Item"
msgstr ""

#: src/debug/tserver/netserver.py:459
msgid "Redirected I/O failed on closed debug connection"
msgstr "Сбой перенаправленного ввода-вывода при закрытом отладочном соединении"

#: src/debug/tserver/netserver.py:613
msgid "EOF when reading a line"
msgstr "EOF при чтении строки"

#: src/debug/tserver/netserver.py:1110
msgid "(unavailable)"
msgstr "(недоступен)"

#: src/debug/tserver/netserver.py:1366
msgid "[omitting some output]"
msgstr "[опуская некоторый вывод]"

#: src/debug/tserver/startdebug.py:195 src/debug/tserver/startdebug.py:373
#, fuzzy
msgid "wingdb.py: Error: Debug file does not exist:"
msgstr "Предупреждение: файл журнала не существует:"

#: src/debug/tserver/startdebug.py:199 src/debug/tserver/startdebug.py:377
#, fuzzy
msgid "wingdb.py: Error: Debug file is not a file:"
msgstr "Предупреждение: файл журнала не является файлом:"

#: src/debug/tserver/startdebug.py:426
msgid "Network peer is "
msgstr ""

#: src/debug/tserver/startdebug.py:427
msgid "Attach port = "
msgstr ""

#: src/debug/tserver/startdebug.py:428
msgid "Initial stop = "
msgstr ""

#: src/debug/tserver/startdebug.py:442
msgid "wingdb.py: Could not create debug server"
msgstr ""

#: src/debug/tserver/startdebug.py:464
msgid "wingdb.py: Running %s"
msgstr ""

#: src/debug/tserver/startdebug.py:475
msgid "wingdb.py: Server exiting abnormally on exception"
msgstr ""

#: src/debug/tserver/startdebug.py:635
msgid "wingdb.py: Error: Failed to start the debug server"
msgstr ""

#: src/debug/tserver/startdebug.py:636
msgid "wingdb.py: Error: You may be running an unsupported version of Python"
msgstr ""

#: src/debug/tserver/startdebug.py:637
msgid "wingdb.py: Python version = %s"
msgstr ""

#: src/debug/tserver/startdebug.py:717 src/debug/tserver/startdebug.py:719
msgid "-- Type return or enter to exit --\n"
msgstr ""
